docs/configuration/yazi.md

```diff
@@ -105,7 +105,7 @@ Available parameters are as follows:
   - `linux`: Linux
   - `macos`: macOS
 
-## open
+## [open]
```

Comment:
Why is there red text above and below this?

Reply:
Maybe I merged it incorrectly, let me check

Reply:
I didn't find any syntax issues, it seems just a bug on GitHub

Reply:
I think its an issue with the Git merge...

Reply:
let's just ignore it!

Reply:
Does it look fine when you build it?

Reply:
yeah
![CleanShot 2024-01-06 at 09 01 05](https://github.com/yazi-rs/yazi-rs.github.io/assets/17523360/2cb33fe1-e444-4d65-8235-dd1afae50969)